omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -71,9 +83,11 @@
     #   /compliance/organizations/ORG/owners/OWNER/compliance[/PROFILE]
     location ~ (?:/compliance)?/organizations/([^/]+)/owners/([^/]+)/compliance(.*) {
       set $request_org $1;
-      set $new_uri "/compliance/profiles/$2$3";
-      access_by_lua_file '/var/opt/opscode/nginx/etc/scripts/check_token_using_endpoint_compliance.lua';
-      proxy_pass $upstream;
+      access_by_lua_block { validator.validate("GET") }
+      proxy_set_header x-data-collector-token $data_collector_token;
```

Comment:
the DC token is used by Automate the authenticate the request:

```
[root@ap-ca6 ~]# curl -k "https://localhost/compliance/profiles"
<html>
<head><title>401 Authorization Required</title></head>
<body bgcolor="white">
<center><h1>401 Authorization Required</h1></center>
<hr><center>nginx</center>
</body>
</html>

[root@ap-ca6 ~]# curl -k "https://localhost/compliance/profiles" -H "x-data-collector-auth: version=1.0" -H "x-data-collector-token: $AUTOMATE_DC_TOKEN" -H "chef-delivery-enterprise: default"
{"admin":{"linux-baseline":{"id":"linux-baseline","owner_id":"admin","owner":"admin","name":"linux-baseline","title":"DevSec Linux Security Baseline","maintainer":"Hardening Framework Team","copyright":"Hardening Framework Team","copyright_email":"hello@hardening.io","license":"Apache 2 license","summary":"Test-suite for best-preactice Linux OS hardening","version":"2.0.1"}}}
```

The use-cases that I remember:
* chef-server -> Automate auth 
* chef-client solo -> Automate auth